presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1153,4 +1154,19 @@ public void testCreateSchema()
         assertUpdate("DROP SCHEMA test_schema_create");
         assertQueryFails("DROP SCHEMA test_schema_create", "line 1:1: Schema '.*\\.test_schema_create' does not exist");
     }
+
+    @Test
+    public void testInsertForDefaultColumn()
```

Comment:
Please add a javadoc to ATDQ and ATIST governing what goes where.

Also, now the test is no longer run for: redis, jdbc+h2, kafka, thrift, ES, Kudu, Accumulo and Mongo